lib/bosh/gen/generators/package_docker_image_generator/templates/jobs/%job_name%/templates/bin/install_ctl.tt

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -e # exit immediately if a simple command exits with a non-zero status
+set -u # report the usage of uninitialized variables
+
+# Setup env vars and folders for the webapp_ctl script
+source /var/vcap/jobs/<%= job_name %>/helpers/ctl_setup.sh '<%= job_name %>'
+
+export PORT=${PORT:-5000}
+export LANG=en_US.UTF-8
+
+case $1 in
+
+  start)
+    pid_guard $PIDFILE $JOB_NAME
+
+    # store pid in $PIDFILE
+    echo $$ > $PIDFILE
+
+    time docker \
```

Comment:
Just to get some feedback on how long the unpacking took. I saw a 500Mb tgz take 2 mins to install.